src/main/java/com/example/bookshop/controller/AuthenticationController.java

```diff
@@ -18,7 +17,6 @@
 @RequestMapping(value = "/api")
```

Comment:
```suggestion
@RequestMapping("/auth")
```

Add `server.servlet.context-path=/api` into application.properties